.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Documentation Request
+about: Suggest an idea for improving the project documentation
+title: '[DOC]'
+labels: doc
+assignees: ''
+
+---
+
+<!--- Provide a general summary of the documentation request in the Title above -->
+<!--- Check existing open and closed documentation requests to see if your idea has been suggested before -->
+
+## Environment Description
+
+<!--- Include as many relevant details about the environment you are working in -->
+* Presto version used:
+* Data source and connectors or catalogs used:
+* Deployment (Cloud or On-prem):
+
+
+## Description
+<!--- Describe where the existing documentation is lacking or unclear -->
+<!--- If suggesting a change/improvement, explain the difference from current behavior -->
+
+## Desired Content
+<!--- Suggest what specific content should be added or updated -->
+
+## Presto Component, Service, or Connector
+<!--- Tell us to which service or component this request is related to -->
+
+## Testing Documentation
+<!--- Provide a set of steps to update the documentation, if applicable -->
+1.
+2.
+
```

Comment:
Suggest adding a "Screenshots (if appropriate)" section, similar to the section in [bug-report.md template](https://github.com/prestodb/presto/blob/master/.github/ISSUE_TEMPLATE/bug-report.md). 

OR, add text, either in the general summary at the beginning or in the Description section, prompting the person opening the issue to provide screenshots of typos or formatting errors in the existing doc, or other screenshots that help explain the problem described in this issue.